addons/binding/org.openhab.binding.folderwatcher/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,126 @@
+                <default>false</default>
+                <description>Allow listing of sub folders</description>
+                <advanced>true</advanced>
+            </parameter>
+            <parameter name="connectionTimeout" type="integer" required="true" min="1">
+                <label>Connection timeout, s</label>
+                <description>Connection timeout for FTP request</description>
+                <default>30</default>
+                <advanced>true</advanced>
+            </parameter>
+            <parameter name="pollInterval" type="integer" required="true" min="1">
+                <label>Polling interval, s</label>
+                <description>Interval for polling folder changes</description>
+                <default>60</default>
+                <advanced>true</advanced>
+            </parameter>
+            <parameter name="diffHours" type="integer" required="true" min="1">
+                <label>Time stamp difference, h</label>
+                <description>How many hours back to analyze</description>
+                <default>24</default>
+                <advanced>true</advanced>
+            </parameter>
+		</config-description>
+
+	</thing-type>
+
+	<channel-type id="newfile-channel">		
+		<kind>trigger</kind>
+		<label>New file name(s)</label>
+		<description>A new file name</description>
+        <category>String</category>
+		<state readOnly="true"></state>		
+	</channel-type>
+
+	<thing-type id="localfolder">
+        <label>Local Folder</label>
+        <description>Local folder to be watched</description>
+
+        <channels>
+            <channel id="newlocalfile" typeId="newfile-channel" />
+        </channels>
+
+        <config-description>
+            <parameter name="localDir" type="text" required="true">
+                <label>Local Directory</label>
+                <description>Local directory to be watched</description>
+            </parameter>       
+            <parameter name="pollIntervalLocal" type="integer" required="true" min="1">
+                <label>Polling interval, s</label>
+                <description>Interval for polling folder changes</description>
+                <default>60</default>
+                <advanced>true</advanced>
+            </parameter>   
+            <parameter name="listHiddenLocal" type="boolean" required="true">
+                <label>List Hidden</label>
+                <default>false</default>
+                <description>Allow listing of hidden files</description>
+                <advanced>true</advanced>
+            </parameter>
+            <parameter name="listRecursiveLocal" type="boolean" required="true">
+                <label>List Sub Folders</label>
+                <default>false</default>
+                <description>Allow listing of sub folders</description>
+                <advanced>true</advanced>
+            </parameter>
+        </config-description>
+    </thing-type>
+</thing:thing-descriptions>
```

Comment:
Have a look for missing newlines. You will see them as icon in the files changed tab here on github.